src/icons/style.css

```diff
@@ -1,6 +1,10 @@
 @font-face {
   font-family: 'icomoon';
-  src: url('./icomoon.woff?ookznz') format('woff');
```

Comment:
The `./` path is _magic_ in Next.js, which is why we were inlining the assets (dumping the file contents as base64 directly into the CSS).

Because this increased our CSS file size, we had an extra step for updating icons where you'd have to remove all font assets that weren't `woff`.

Now that the icon font assets are no longer inlined, we can include all of them in the `src` stack.